newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/PointCutClassTransformer.java

```diff
@@ -381,7 +381,8 @@ public WeavingLoaderImpl(ClassLoader classLoader) {
         private void recordSupportabilityMetrics(List<PointCut> appliedPointCuts) {
             for (PointCut pointCut: appliedPointCuts) {
                 ServiceFactory.getStatsService().doStatsWork(StatsWorks.getRecordMetricWork(MessageFormat.format(
-                        MetricNames.SUPPORTABILITY_POINTCUT_LOADED, pointCut.getName()), 1));
+                        MetricNames.SUPPORTABILITY_POINTCUT_LOADED, pointCut.getName()), 1), MetricNames.SUPPORTABILITY_POINTCUT_LOADED +
+                        " " + pointCut.getName());
```

Comment:
The log message for this entry will look odd:
Supportability/PointCutInstrumentation/Loaded/{0} pointcutName


As a general rule, when the metric name is created using String/Message.format, so should the statsWorkName.
Easiest way to do this is to create a var with the metric name and using it again as the statsWorkName.

Reply:
done